### DISCUSSION TITLE
Open "Save Configuration" via keyboard shortcut

### DISCUSSION THREAD

**Discussion by GioAc96** [mia-platform/community#118] [Ideas]
I find myself using the `CRL+K` and `CTRL+ENTER` keyboard shortcuts all the time when using the Console. However, I really miss a keyboard shortcut that directly opens the "save configuration" modal. Would you consider implementing this feature? I think that `CTRL+S` (`CMD+S` on Mac) would be a fitting candidate

**Comment by davidebianchi** [comment]
Hi @GioAc96! Sure, we want to add also other shortcut to the already supported. In the next version, we add some shortctut (to speed up some main actions) and we also continue to add it. The save is one of the shortcut we want to add, but it's a bit more complex from the UX perspective as the modal is on a popover and we already thought about it.
